### PR TITLE
2.16 release notes and highlights

### PR DESCRIPTION
**Pull request by naemono**
Release notes and highlights for the upcoming 2.16.0 release.

Preview: https://cloud-on-k8s_bk_8301.docs-preview.app.elstc.co/guide/en/cloud-on-k8s/master/release-notes-2.16.0.html

### CODE REVIEW

File: docs/release-notes.asciidoc
Comment (pebrc) [inline docs/release-notes.asciidoc:1]: I think you missed to link up the highlights to its index

Comment (naemono) [reply docs/release-notes.asciidoc:1]: You're absolutely right. Added it.

File: docs/release-notes/2.16.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+[float]
+=== New features
+
+* Support remote clusters using API keys {pull}8089[#8089] (issues: {issue}7818[#7818], {issue}8167[#8167])
+
+[[enhancement-2.16.0]]
+[float]
+=== Enhancements
+
+* Make helm charts consistent with how fields in spec are handled in eck-fleet-server. {pull}8285[#8285]
```
Comment (pebrc) [inline docs/release-notes/2.16.0.asciidoc:19]:
```suggestion
* Make Helm charts consistent in how fields in spec are handled (eck-fleet-server). {pull}8285[#8285]
```
Capital initial for Helm: I am sure we should do that for consistency
The rewording, not 100% as I am no native speaker, but something in the original sentence is a bit odd.

**Review by pebrc — APPROVED** [review]
LGTM one more susggestion for the Helm chart PRs

File: docs/release-notes/2.16.0.asciidoc (hunk 1 above)
Comment (naemono) [reply docs/release-notes/2.16.0.asciidoc]: > something in the original sentence is a bit odd.

I don't disagree. I've attempted a re-wording of this. Lmk your thoughts @pebrc

**Review by pebrc — APPROVED** [review]

**Pull request merged**